_inc/client/writing/composing.jsx

```diff
@@ -233,6 +235,29 @@ export class Composing extends React.Component {
 					</FormFieldset>
 				</SettingsGroup>
 			),
+			wordPressComComposeSettings = (
+				<SettingsGroup
+					module={ wordPressComCompose }
+					support={ {
+						text: __( 'Allow new block editor posts to be composed on WordPress.com.' ),
+						link: 'https://jetpack.com/support/copy-post-2/',
```

Comment:
I assume this link will be updated to something else if we finally release this :)

Reply:
Sadly, no, that's the real slug (already published and live) :/. There was a reason I can't seem to find, maybe @jeherve remembers?

Reply:
It can happen because the slug without the `-2` is already used by another page on the site, so it can't be re-used. In this specific case, the slug is used by a media item.

Reply:
Oh! Duh, right – I blame morning 🙃

Reply:
Updated in c8e0a2e3c.